tests/test_api.py

```diff
@@ -46,6 +46,10 @@ def postmaster_start_time():
     def replica_cached_timeline(_):
         return 2
 
+    @staticmethod
+    def is_running():
+        return True
```

Comment:
Why don't we mock this call? We are sure we will not need to return False in this suit?

Reply:
We mock the entire Postgresql.

Reply:
Oh, I see. Sorry  🤦‍♀️